native/tests/src/test_persistence.cpp

```diff
@@ -2,6 +2,7 @@
 #include <mixpanel/mixpanel.hpp>
 #include <mixpanel/detail/persistence.hpp>
 #include <thread>
+#include <fstream>
```

Comment:
I mostly agree with the thoughts [here](https://blog.knatten.org/2012/11/30/how-to-avoid-includes-in-headers/)